pkgs/top-level/php-packages.nix

```diff
@@ -254,14 +255,14 @@ let
     buildInputs = [ (if isPhp73 then pkgs.pcre2 else pkgs.pcre) ];
   };
 
-  pcs = buildPecl {
+  pcs = assert !isPhp74; buildPecl {
     version = "1.3.3";
     pname = "pcs";
 
     sha256 = "0d4p1gpl8gkzdiv860qzxfz250ryf0wmjgyc8qcaaqgkdyh5jy5p";
   };
 
-  pdo_sqlsrv = buildPecl {
```

Comment:
Assertions prevent overriding. Do we have something like `enabled` attribute from Python modules? Otherwise we would need `meta.broken`.

Also it would be nice to add comennts explaining the disabling.

Reply:
in the master branch these packages are builded
```
nix build -f ./default.nix php72Packages.pdo_sqlsrv php73Packages.pdo_sqlsrv
ls -lah | grep sqlsrv
lrwxrwxrwx  1 user user   64 ноя 25 11:55 result -> /nix/store/cl38liqhl1kdrrrvgyg9npp0k08dn5bd-php-pdo_sqlsrv-5.6.1
lrwxrwxrwx  1 user user   64 ноя 25 11:55 result-1 -> /nix/store/iy5ii6dinijd1hl03y08giin3350mfy0-php-pdo_sqlsrv-5.6.1
```

Reply:
Using `meta.broken = isPhp74;` in the argument set to `buildPecl` instead should work.

Reply:
Fixed :)

Reply:
Could you add the comments explaining the disabling?